src/opnsense/mvc/app/models/OPNsense/Firewall/Menu/Menu.xml

```diff
@@ -1,17 +1,17 @@
 <menu>
     <Firewall>
-        <Automation cssClass="fa fa-gear">
-            <Filter order="50" url="/ui/firewall/filter/">
+        <Rules>
+            <Filter order="0" VisibleName="Filter [new]" url="/ui/firewall/filter/">
```

Comment:
I think this already starts at 0, so making a bit of room over there is better:

https://github.com/opnsense/core/blob/5256bed259b194c5f3d16f00eb215fbc78bb11e6/src/opnsense/mvc/app/models/OPNsense/Base/Menu/MenuSystem.php#L333